src/server/routes/authenticator.js

```diff
@@ -4,14 +4,20 @@
 
 const jwt = require('jsonwebtoken');
 const secretToken = require('../config').secretToken;
+const validate = require('jsonschema').validate;
 
 /**
  * Middleware function to force a route to require authentication
  * Verifies the request's token against the server's secret token
  */
 module.exports = (req, res, next) => {
 	const token = req.headers.token || req.body.token || req.query.token;
-	if (token) {
+	const validParams = {
+		type: 'string',
+	};
+	if (!validate(validParams, token).valid) {
```

Comment:
You have validParams and token backwards. This causes all authenticated routes to fail.